style.css

```diff
@@ -0,0 +1,67 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  box-sizing: border-box;
+  font-family: sans-serif;
+  font-size: 1.6rem;
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+}
+
+#container {
+  background: rgb(207, 247, 183);
+}
+
+header h1 {
+  font-size: 4rem;
+  padding-top: 30px;
+}
+
+header h2 {
+  font-size: 1.5rem;
+}
+
+.image {
+  background: white;
```

Comment:
- [ ] I think in your next project its better to write the color using HEX values rather than the real color name.